.github/workflows/build_windows.yml

```diff
@@ -0,0 +1,59 @@
+name: Build (Windows)
+
+on: [push, pull_request]
+
+jobs:
+  Windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        configuration: ["Debug", "Release"]
+    steps:
+    - name: Define Build Options
+      shell: python
+      run: |
+        if "${{ matrix.configuration }}" == "Debug":
+          print('Configuring for Debug')
+          print('::set-env name=BUILD_PARAM::')
+          print('::set-env name=ARTIFACT_NAME::xqemu-debug')
+        else:
+          print('Configuring for Release')
+          print('::set-env name=BUILD_PARAM::--release')
+          print('::set-env name=ARTIFACT_NAME::xqemu-release')
+    - name: Clone Tree
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Install Dependencies
+      run: |
+        echo "Downloading MSYS2 environment..."
+        Invoke-WebRequest -Uri "https://github.com/mborgerson/msys2-tools/releases/latest/download/msys64.7z" -OutFile "msys64.7z"
```

Comment:
I think it's foreseeable that this PR will be merged.
Please move `mborgerson/msys2-tools` to `xqemu/ci-environments` or something.

I'm not happy about either of those repo names; I'd like something extensible in case we also create environments for other toolchains. The release is already called msys64.7z anyway (which means we could also add other environments to that repository).

If we do one repository per environment, the repo name should probably be along the lines of `ci-environment-msys64` (or `ci-environment-msys2` if we decide to share it for 32 bit?)

Reply:
I accept the `ci-environment-msys2` proposal. Repo has been moved, and the PR has been updated.